README.md

```diff
@@ -80,7 +81,7 @@ Additional credits for bug fixes, localisations and suggestions go to:
  * National liturgical calendars for country specific calendars.
  * Richly commented code to help developers and contributors understand how the module works.
 
-NOTE: This module uses [Moment](http://momentjs.com/) and [lodash](http://lodash.com/) for calculations and operations. Additional plugins such as [Range](https://github.com/gf3/moment-range) and [Recur](https://github.com/c-trimm/moment-recur) are used to extend date computation functionality. 
+NOTE: This module uses [Moment](http://momentjs.com/) and [lodash](http://lodash.com/) for calculations and operations. Additional plugins such as [Range](https://github.com/gf3/moment-range) and [Recur](https://github.com/c-trimm/moment-recur) are used to extend date computation functionality.
```

Comment:
I think we should use one name for Moment/Moment.js. Currently we use difference names. The Moment website uses _moment_, but that is quite ambigious.

And Lodash (as a name) should be written with capital initial letter.

Reply:
I've added a capital everywhere in the edited sources, so it's more explicit that it refer to the Moment library. I only added the .js when it could be ambiguous (when Moment is the first word of a phrase). Otherwise I find it too verbose to specify the .js everywhere.